### PR TITLE
[fix bug 1283397] Decommission Pocket pages

### PR DESCRIPTION
**Pull request by alexgibson**
## Description
- Removes old `/whatsnew` pages relating to Pocket and the `firefox/pocket` feature page.

## Bugzilla link
https://bugzilla.mozilla.org/show_bug.cgi?id=1283397

## Testing
Check to make sure there are no stray file references left for Pocket assets or URLs. I've added a test for the new redirect.

## Checklist
- [x] Requires l10n changes.
- [x] Related functional & integration tests passing.

### CODE REVIEW

**Comment by alexgibson** [comment]
@Peiying - I've marked this as requiring l10n changes as there are strings no longer being used once these pages are removed. I did notice some of the current `/firstrun` pages seem to be importing the `firefox/whatsnew_38` lang file still. The history of why here is lost on me, but I'm happy to defer to what needs doing here (if anything).

**Comment by flodolo** [comment]
After creating firefox/whatsnew_38.lang for 38.0.5 firstrun/whatsnew, we kept using that for all updates. That's expected at this point.

**Comment by alexgibson** [comment]
> After creating firefox/whatsnew_38.lang for 38.0.5 firstrun/whatsnew, we kept using that for all updates. That's expected at this point.

OK, will keep references to that lang file around. Thanks @flodolo

File: bedrock/firefox/views.py [hunk 1]
```diff
@@ -438,17 +406,6 @@ def get_template_names(self):
...
-            has_video = LOCALE_SPRING_CAMPAIGN_VIDEOS.get(locale, False)
-            has_pocket = locale in self.pocket_locales
-            if has_pocket and has_video:
-                template = 'firefox/whatsnew_38/whatsnew-pocket-video.html'
-            elif has_video:
-                template = 'firefox/whatsnew_38/whatsnew-video.html'
-            elif has_pocket:
-                template = 'firefox/whatsnew_38/whatsnew-pocket.html'
-            else:
-                template = 'firefox/australis/whatsnew.html'
```
Comment (jpetto) [inline bedrock/firefox/views.py]: These lines right here, these are the *best* deletions. ⭐

File: bedrock/firefox/views.py [hunk 2]
```diff
@@ -51,16 +51,6 @@
...
 
-LOCALE_SPRING_CAMPAIGN_VIDEOS = {
-    'en-US': 'https://videos.cdn.mozilla.net/uploads/marketing/SpringCampaign2015/Firefox_Welcome_english',
-    'en-GB': 'https://videos.cdn.mozilla.net/uploads/marketing/SpringCampaign2015/Firefox_Welcome_englishUK',
-    'de': 'https://videos.cdn.mozilla.net/uploads/marketing/SpringCampaign2015/Firefox_Welcome_german',
-    'es-ES': 'https://videos.cdn.mozilla.net/uploads/marketing/SpringCampaign2015/Firefox_Welcome_spanish',
-    'es-MX': 'https://videos.cdn.mozilla.net/uploads/marketing/SpringCampaign2015/Firefox_Welcome_spanishMX',
-    'fr': 'https://videos.cdn.mozilla.net/uploads/marketing/SpringCampaign2015/Firefox_Welcome_french',
-    'pt-BR': 'https://videos.cdn.mozilla.net/uploads/marketing/SpringCampaign2015/Firefox_Welcome_portugeseBrazil',
-}
```
Comment (jpetto) [inline bedrock/firefox/views.py]: Should we file a bug to remove these videos from the CDN?

**Comment by jpetto** [comment]
LGTM! r+

File: bedrock/firefox/views.py (hunk 2 above)
Comment (alexgibson) [inline bedrock/firefox/views.py]: Good call, will do so once I figure out what component it should be :)

**Pull request merged**